app/templates/package.json

```diff
@@ -23,27 +23,29 @@
   },
   "homepage": "https://github.com/<%= user %>/<%= repo %>",
   "devDependencies": {
-    "babel-core": "^5.2.17",
-    "babel-eslint": "^4.0.5",
-    "babel-loader": "^5.3.2",
-    "chai": "^3.2.0",
+    "babel-core": "^6.3.26",
```

Comment:
@jmm, do you happen to know if `babel-core/register` is deprecated? I recall reading that `babel-register` is preferable, but it ends up being duplicate code to install that as well as `babel-core`, given that `babel-core/register` exists. I think that's where @megawac was coming from.